packages/shared/package.json

```diff
@@ -38,7 +38,6 @@
   },
   "devDependencies": {
     "@jest/globals": "^29.7.0",
-    "@logto/connector-kit": "workspace:^3.0.0",
```

Comment:
just double check, no connector-kit is used in this package?

Reply:
Yes, checked.